Website/htdocs/mpmanager/database/factories/PersonFactory.php

```diff
@@ -17,7 +17,7 @@ public function definition(): array
             'surname' => $this->faker->firstName(),
             'birth_date' => $this->faker->date(),
             'sex' => $this->faker->randomKey(['male', 'female']),
-            'is_customer' => 1,
+            'is_customer' => 0,
```

Comment:
@buralp  What was the reason to change that?

Reply:
because i want to check that is customer in PersonObserver.php